src/styles/colors.tsx

```diff
@@ -39,6 +39,9 @@ export enum Colors {
   successDark = '#137211',
   successLight = '#F1FDF1',
 
+  informationDark = '#0768AE',
```

Comment:
we shouldn't be adding new colors to the stylesheet unless it exists in the design system. there is already an informational and onboardingBlue using this hex value. in figma it looks like it says onboardingBlue when I inspect the value, so please use that instead

Reply:
## Primary colour
[Icon](https://www.figma.com/file/l6cnlCx2Ht2YSgnbBdzZph/App-Style-Guide?type=design&node-id=311-269&mode=design&t=RCaFcCeXk1L63S33-4) uses `background: #0768AE;` (Onboarding/Blue) matching [Info-Dark](https://www.figma.com/file/erFfzHvSTm5g1sjK6jWyEH/Working-Design-System-Components?type=design&node-id=213-4&mode=design&t=qWFasC1gqiWu4zR0-4)

We already had informational and onboardingBlue. I'll rename `informational` to `informationDark`.